packages/block-library/src/navigation-link/link-ui.js

```diff
@@ -301,8 +289,29 @@ const LinkUITools = ( { setAddingBlock, focusAddBlockButton } ) => {
 		}
 	}, [ focusAddBlockButton ] );
 
+	// Focus the add page button when the popover is opened.
+	useEffect( () => {
+		if ( focusAddPageButton ) {
+			addPageButtonRef.current?.focus();
+		}
+	}, [ focusAddPageButton ] );
+
 	return (
-		<VStack className="link-ui-tools">
+		<VStack spacing={ 0 } className="link-ui-tools">
+			{ canCreatePage && (
+				<Button
+					__next40pxDefaultSize
+					ref={ addPageButtonRef }
+					icon={ page }
+					onClick={ ( e ) => {
+						e.preventDefault();
+						setAddingPage( true );
+					} }
+					aria-haspopup={ blockInserterAriaRole }
+				>
+					{ __( 'Add page' ) }
```

Comment:
```suggestion
					{ __( 'Create page' ) }
```
I think "Create" makes more sense here - "Adding" is the process of putting the link into the navigation, "Creating" is the process of actually making the entity.

Reply:
I still think "Create" is the right language here.

Reply:
Oh man sorry I missed this. I'll follow up now.

Reply:
https://github.com/WordPress/gutenberg/pull/71487